src/tests/unit/tests/reports/components/report-sections/__snapshots__/collapsible-url-result-section.test.tsx.snap

```diff
@@ -4,10 +4,6 @@ exports[`CollapsibleUrlResultSection renders 1`] = `
 <DocumentFragment>
   <div
     class="urlResultSection"
-  >
-    <div>
-      Collapsible component
-    </div>
-  </div>
+  />
```

Comment:
We should either prioritize finding a way for this content to make it into the snapshot or at minimum ensure that this content was retrieved by making the `collapsibleControlMock` call `verifiable` and calling `collapsibleControlMock.verifyAll()` at the end of the test.

Reply:
Updated the snapshot file.